webapp/src/components/common/musicplayer/component/LibraryPlayer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import ReactPlayer from "react-player";
+
+
+export interface LibraryPlayerProps {
+    selectedFile?: Blob
+}
+
+
+
+export default function LibraryPlayer(props?: LibraryPlayerProps) {
+
+
+    if (!props || !props.selectedFile) {
+        return (<div>Please select a lick to play</div>);
+    }
+
+    let audioPath = "";
+    if (props.selectedFile) {
+        try {
+            audioPath = URL.createObjectURL(props.selectedFile);
+        } catch {
+            return (<div>Failed to play the lick</div>);
+        }
+    }
+    if (audioPath) {
+        return (
+            <ReactPlayer url={audioPath}
```

Comment:
nice! this works great, would be nice if in the future playing the audio was embedded into the table, like each lick has their own play button, i know you're probably thinking that already too